fv/wireguard_test.go

```diff
@@ -351,6 +329,26 @@ var _ = infrastructure.DatastoreDescribe("WireGuard-Supported", []apiconfig.Data
 			Expect(rcvd[1]).NotTo(BeEmpty())
 			Expect(rcvd[1]).To(Equal(sent[0]))
 		})
+
+		It("between pod to pod should be encrypted using wg tunnel with egress policy applied", func() {
```

Comment:
How do you know the policy is doing what you'd expect?

I think you should also have a similar test where the policy is set to "Deny" and you check the packet counts drop to 0.  Perhaps have it as part of the same test, so you modify the existing policy and then retry.

Reply:
I had both tests initially but removed the `deny` test thinking `allow` test will be sufficient to invoke the policy code-path. Added both `allow` and `deny` tests now.